### ISSUE TITLE
nl.yml

### ISSUE THREAD

**Issue by tomhughes** [tomhughes/trac-tickets#1911]
**[Submitted to the original trac issue database at 5.54pm, Wednesday, 3rd June 2009]**

I created a Dutch translation. Almost everything has been translated. Because I don't have access to svn, I put it here; could someone put it in the svn?

**Comment by tomhughes** [comment]
**[Added to the original trac issue at 5.56pm, Wednesday, 3rd June 2009]**

(In [15576]) Add Dutch translation. Closes #1911.

**Comment by tomhughes** [comment]
**Author: Willem1**
**[Added to the original trac issue at 6.02pm, Wednesday, 3rd June 2009]**

Replying to [comment:1 tomhughes]:
> (In [15576]) Add Dutch translation. Closes #1911.
Seems there is a problem... when I tried it I got

'''Application error

The OpenStreetMap server encountered an unexpected condition that prevented it from fulfilling the request (HTTP 500)

Feel free to contact the OpenStreetMap community if your problem persists. Make a note of the exact URL / post data of your request.

This may be a problem in our Ruby On Rails code. 500 ocurrs with exceptions thrown outside of an action (like in Dispatcher setups or broken Ruby code)
'''
